src/webserver.cc

```diff
@@ -121,7 +129,7 @@ void Webserver::runThread(int threadIndex) {
             processConnection(threadIndex, socket);
 
             // rate limit inbound connections
-            std::this_thread::sleep_for(std::chrono::milliseconds(100));
+            //std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
So I believe we are officially getting rid of this if we cannot find a use for it. Do we need #include <chrono>